WolvenKit.App/ViewModels/Tools/ProjectExplorerViewModel.cs

```diff
@@ -554,7 +556,9 @@ private void CopyAbsPathToCurrentFolder() =>
     /// <summary>
     /// Reimports the game file to replace the current one
     /// </summary>
-    private bool CanAddDependencies() => ActiveProject != null && SelectedItem != null && IsInRawFolder(SelectedItem) && SelectedItem.Extension.ToLower().Contains("xml");
+    private bool CanAddDependencies() => ActiveProject != null && SelectedItem?.IsInRaw == true &&
```

Comment:
Rather use the existing IsInRawFolder methods

Reply:
I moved these into properties of the model, seemed more object oriented that way :) any reason for that?

Reply:
Yeah the reason is that the "is in raw" check should check if the path is inside the project folder, and not just a substring check

Reply:
that stuff was super confusing, because the `relativeXXX` paths were different depending on whether the item is in raw or not, so the logic was flakey here and didn't work like I expected it :|